microsite/pages/en/index.js

```diff
@@ -95,7 +95,7 @@ class Index extends React.Component {
                 src={`${baseUrl}animations/backstage-service-catalog-icon-1.gif`}
               />
               <Block.Subtitle>
-                Backstage Service Catalog{' '}
+                [Backstage Service Catalog](#backstage-service-catalog)
```

Comment:
These needs to be regular HTML links (and not Markdown formated). Example:

```
<a href="#backstage-service-catalog">the link title</a>
```

Reply:
Okay, gave it another shot. Did that work?